coding-challenges.js

```diff
@@ -1,60 +1,369 @@
+        "",
+        "_____________________________________",
+        "_____________________________________",
+        "", "",
+        "        Altered Back-to-Back",
+        "", "",
+        `${BLACK_BACKGROUND}${GREEN_TEXT}  < Round One >  ${FORMAT_RESET}`,
+        //" < Round One >",
+        "-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-",
+        alteredArr_1,
+        alteredArr_2,
+        alteredArr_3,
+        alteredArr_4,
+        alteredArr_5,
+        alteredArr_6,
+        "-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-",
+        "", "",
+        `${BLACK_BACKGROUND}${GREEN_TEXT}  < Round Two >  ${FORMAT_RESET}`,
+        //" < Round Two >",
+        "-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-",
+        alteredArr__1,
+        alteredArr__2,
+        alteredArr__3,
+        alteredArr__4,
+        alteredArr__5,
+        alteredArr__6,
+        "-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-*-"
+
+    )
+
+/// - - - - - - - - - - - - - - - -
+/// - - - - - - - - - - - - - - - -
+
+
+
+
+
+
+
+//  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /
+
+// --------------------2)   Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
+
+//  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /
+
+var cubeAndSum1 = [2, 3, 4]     // Expected output: 99
+var cubeAndSum2 = [0, 5, 10]    // Expected output: 1125
+var cubeAndSum3 = [4, 7, 8]     // Expected output: 919     >>     My own example
+
+
+const cubeAndSum = numArrIn => {
+
+    return numArrIn.reduce( (accumulator, currentValue) => accumulator + currentValue**3, 0 )
+    // the first element in the array would have been skipped and used as the ' initial value ' if I did not specify to use ' 0 ' as that value.
+}
+
+
+
+/// - - - - - - - - - - - - - - - -
+
+log(    "Cube All Numbers in an Array, Then Add Them Up",
+        
+        `[${cubeAndSum1}]   >>   ${cubeAndSum( cubeAndSum1 )}`,
+        `[${cubeAndSum2}]  >>   ${cubeAndSum( cubeAndSum2 )}`,
+        `[${cubeAndSum3}]   >>   ${cubeAndSum( cubeAndSum3 )}            (my own example)`
+
+    )
+
+/// - - - - - - - - - - - - - - - -
+
+
+
+
+
+//  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /
+
+// --------------------3)   Create a function that takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
+
+//  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /  / /
+
+var nums1 = [3, 56, 90, -8, 0, 23, 6]   // Expected output: [-8, 90]
+var nums2 = [109, 5, 9, -59, 8, 24]     // Expected output: [-59, 109]
+
 
 
+const getMinAndMax = numArrIn => {
+    return numArrIn.sort( (a,b) => a-b ).filter( (v,i,arr) => i<1 || i===arr.length-1 )
+}
```

Comment:
i really love all of the optimized code, but just be sure to comment your code. especially when it's not immediately obvious what your code is doing.